Source/SwiftyDropbox/Shared/Handwritten/OAuth/AuthSession.swift

```diff
@@ -0,0 +1,104 @@
+///
+/// Copyright (c) 2020 Dropbox, Inc. All rights reserved.
+///
+
+import Foundation
+import CommonCrypto
+
+// MARK: Public
+
+/// Struct contains the information of a requested scopes.
+public struct ScopeRequest {
+    /// Type of the requested scopes.
+    public enum ScopeType: String {
+        case team
+        case user
+    }
+
+    let scopes: [String]
+    let includeGrantedScopes: Bool
+    let scopeType: ScopeType
+
+    var scopeString: String? {
```

Comment:
its unclear to me what a "scopeString" should be? its used in "scope" in the url but could you elaborate somewhere?

Reply:
will add doc string to this field.